code/game/objects/structures/signs/signs_plaques.dm

```diff
@@ -23,7 +23,7 @@
 
 /obj/structure/sign/plaques/kiddie
 	name = "\improper AI developers plaque"
-	desc = "Next to the extremely long list of names and job titles, there is a drawing of a little child. The child appears to be retarded. Beneath the image, someone has scratched the word \"PACKETS\"."
+	desc = "Next to the extremely long list of names and job titles, there is a drawing of a little child. The child appears to be low IQ. Beneath the image, someone has scratched the word \"PACKETS\"."
```

Comment:
"low IQ" is kinda meh in terms of being an insult. Can we have autistic instead?